src/cmd/src/lib.rs

```diff
@@ -79,15 +78,6 @@ pub fn log_versions(version_string: &str, app_version: &str) {
     log_env_flags();
 }
 
-pub fn greptimedb_cli() -> clap::Command {
-    let cmd = clap::Command::new("greptimedb").subcommand_required(true);
-
-    #[cfg(feature = "tokio-console")]
-    let cmd = cmd.arg(arg!(--"tokio-console-addr"[TOKIO_CONSOLE_ADDR]));
-
-    cmd.args([arg!(--"log-dir"[LOG_DIR]), arg!(--"log-level"[LOG_LEVEL])])
-}
```

Comment:
We switched to operating `Command` directly in https://github.com/GreptimeTeam/greptimedb/pull/2911

Does this prevent us from extending arguments or using the command in other binaries? @MichaelScofield Please take a look.

Reply:
IMO, some original args(`--log-level`, `--log-dir`, `--tokio-console-addr`) should be global. The PR will take those args global explicitly. We can still extend the args by adding the new fields into the related struct.

Reply:
My intention in #2911 was to create `clap::Command` dynamically, to use different sub commands. So is it possible to make the new `Command` owned a dyn `SubCommand`?

Reply:
What's the scenario of #2911, or how can I test the cli from a user's perspective? I think this PR is equivalent based on #2911.

Reply:
See the codes here https://github.com/GreptimeTeam/greptimedb-enterprise/blob/develop/projects/dynamo/src/bin/greptime.rs

Reply:
ok to me